finishFeature/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="retry">Retry</string>
+  <string name="go_main_menu">Go to main menu</string>
+  <string name="total_score">Total score: %1$d / %2$d</string>
```

Comment:
better make it actionable as code below
`action_retry`
`action_redirect_main_menu`
`...`

if you have many screens in the feature better to add at least partially, in the beginning, to describe to which part it belongs as
`finish_quiz_total_score`

Reply:
+